.forge-snapshots/autocompound_excessFeesCredit.snap

```diff
@@ -1 +1 @@
-311783
\ No newline at end of file
+192327
```

Comment:
is this a real decrease??

Reply:
what do u mean real... a lot of the testing infra changed so could partially be attributed to that too